src/content/caseStudies/contenda-share-pear/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Content for Everyone"
+description: "Building Public-Sharing of Blog Drafts in 1 Month"
```

Comment:
```suggestion
description: "Building Public-Sharing of Blog Drafts in One Month"
```

Reply:
This still needs fixing to read “One” right?

Reply:
Yup, bumping so Darius sees and commits! I thought it got lost in git history or something.